assets/js/functions.js

```diff
@@ -42,8 +42,17 @@ async function trueDarkModeLoop() {
 	setTimeout(function() {
 		chrome.storage.sync.get('np_true_dark', function(data) {
 			var games = data['np_true_dark'];
-			var bool = games[getGameId()] != undefined && games[getGameId()];
-			window.trueDarkMode = bool;
+      if (games != undefined){
+				var bool = games[getGameId()] != undefined && games[getGameId()];
```

Comment:
We could probably just use `var bool = (games != undefined && games[getGameId()] ... )` here, instead of setting it in the get loop. I think it simplifies it a lot.

Also, there's some weird whitespace stuff going on with your editor. 😆